setup.cfg

```diff
@@ -24,7 +24,7 @@ install_requires =
     SQLAlchemy==2.0.16
     grpcio==1.62.1
     gunicorn>=22.0.0
-    numpy
+    numpy<=1.26.0
```

Comment:
should we do 1.26.4? https://numpy.org/doc/stable/release/1.26.4-notes.html

Reply:
1.26.4 also works. updated to 1.26.4